docs/kernel/SystemCallList.md

```diff
@@ -27,14 +27,16 @@ Attempts to get details on the current primary device for a device type. Think o
 
 ### Args:
 - `arg0`: info details. 0 = basic (returned in regs), 1 = advanced (returned as memory block).
-- `arg1`: device type. Corresponds to entries of DeviceType enum found in GenericDevice.h.
+- `arg1`: device type. Corresponds to entries of DeviceType enum found in `libs/np-syslib/include/SyscallEnums.h`.
```

Comment:
i wonder if it could be a good idea to make it a link to the file, so people doesn't have to search for it :D